PWGEM/Dilepton/Core/Dilepton.h

```diff
@@ -946,7 +977,26 @@ struct Dilepton {
       } else if (t1.sign() < 0 && t2.sign() < 0) { // LS--
         fRegistry.fill(HIST("Pair/") + HIST(event_pair_types[ev_id]) + HIST("lsmm/hs"), v12.M(), v12.Pt(), pair_dca, dphi, deta, weight);
       }
-
+    } else if (cfgAnalysisType == static_cast<int>(o2::aod::pwgem::dilepton::utils::pairutil::DileptonAnalysisType::kDCA)) {
+      float deta = v1.Eta() - v2.Eta();
```

Comment:
In case of the `kDCA`, I dont think you need to calculate the dphi and data, right?